app/models/list.rb

```diff
@@ -1,7 +1,14 @@
 class List < ActiveRecord::Base
   belongs_to :therapist
-  has_many :requests
+  has_many :gift_requests
 
   validates :description, presence: true
   validates :title, presence: true
+
+  scope :empty, -> { includes(:gift_requests).where(gift_requests: {id: nil}) }
+  scope :non_empty, -> { joins(:gift_requests) }
+
+  def name
```

Comment:
should be replaced with an `alias` call instead of wrapping it in a method like this